### PR TITLE
Feature/2 setting screen colors

### LINKED ISSUE #2: Setting Screens Colors
# REQUIREMENTS
Making a script to store all the colors need in the app

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.

# Acceptance Criteria
List all required acceptance criteria
- [ ] Create colors scripts
- [ ] Test implementation

### PR DESCRIPTION
**Pull request by NanciGarcia**
# Description
-Created the colors file
-Added color palette
-Added test file
-The color test was carried out for the interfaces
- These changes help to style the components

### CODE REVIEW

**Comment by NanciGarcia** [comment]
Change Sand-box

**Review by JordiGN — APPROVED** [review]
Great selection of the color pallete

**Review by Ocosmee — APPROVED** [review]
Nice job!

File: tribomobile/src/colors.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  BlueLight: '#12a4ed',
+  BlueDark: '#0b0f34',
+  OrangeFonts: '#e98d1e',
+  OrangeDark: '#fe8500',
+  GreenContent: '#7ad520',
+  GrayLight: '#dfe0e5',
+  GrayDark: '#9b9eb1',
+  blueP: '#037d94',
+  mediumvioletred: '#c71585',
+
```
Comment (develiasdaniel) [inline tribomobile/src/colors.js:12]: Remember not to leave blank spaces

**Review by develiasdaniel — APPROVED** [review]
Good work, good that you implemented tests

**Review by Ing-Heber — APPROVED** [review]

**Pull request merged**